src/shared/routes/routes.ts

```diff
@@ -0,0 +1,15 @@
+enum ROUTES {
+  BASE = '/',
+  LOGIN = '/login',
+  ERROR = '/404',
+  BUDDY_ADD_TASK = '/buddy/add-task',
+  BUDDY_TASK_DETAILS = '/buddy/tasks/:taskId',
+  BUDDY_TASKS_LIST = '/buddy/tasks',
+  BUDDY_SELECT_NEWBIE = '/buddy/select-newbie',
```

Comment:
Sure. I've spent most of the time thinking how to organise those IDs. Fine for me.
I've changed routes according to your suggestions here: https://github.com/BuildIt-Poland/buddy-app/pull/12/commits/59a4f90a6cb747116ace75da35b618ff27c6c65a

One question, because I've also think about it (and remove it)
```  NEWBIE_BUDDY_DETAILS = '/newbie/buddy',``` 
This is route for the Newbie, to see Buddy details page (contact information). We don't use ID here `/newbie/:buddyId/buddy`, because Newbie will have only one Buddy and it's not needed, right?

Reply:
Yes, you are totally right 👍 Good job